website/src/components/canvas/Sidebar.jsx

```diff
@@ -104,16 +138,34 @@ class Sidebar extends React.Component {
       <div
       onMouseEnter={this.mouseEnter}
       onMouseLeave={this.mouseLeave}>
-
+       <IconButton
+        iconStyle={styles.arrowOpen}
+        tooltip="Show sidebar"
+        tooltipPosition="bottom-right"
```

Comment:
We don't need a tooltip for this, it's pretty self-explanatory (see how CraftML does the slide-in, slide-out code editor)